src/transformState.js

```diff
@@ -5,7 +5,26 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties': Object.assign(state, action.extraData);
+        break;
```

Comment:
fix indentation everywhere
```suggestion
      case 'addProperties': 
        Object.assign(state, action.extraData);
        break;
```